webapp/loggedin/surveyViews/surveyForm/nodeDefs/components/types/nodeDefEntityFormNodeSelect.js

```diff
@@ -30,13 +31,13 @@ const NodeDefEntityFormNodeSelect = props => {
       <select
         className="node-select"
         value={selectedNode ? Node.getUuid(selectedNode) : 'placeholder'}
-        onChange={e => onChange(e.target.value)}
+        onChange={(e) => onChange(e.target.value)}
         aria-disabled={R.isEmpty(nodes)}
       >
         <option value="placeholder" disabled hidden={true}>
           {i18n.t('surveyForm.nodeDefEntityForm.select')}
         </option>
-        {nodes.map(n => (
+        {nodes.map((n) => (
```

Comment:
node?

Reply:
that component must be refactored when we address views refactor